src/app/qgswelcomepageitemsmodel.cpp

```diff
@@ -44,9 +110,8 @@ QVariant QgsWelcomePageItemsModel::data( const QModelIndex& index, int role ) co
   switch ( role )
   {
     case Qt::DisplayRole:
-      return mRecentProjects.at( index.row() ).title;
+      return QString( "<span style='font-size:18px;font-weight:bold;'>%1</span><br>%2" ).arg( mRecentProjects.at( index.row() ).title != mRecentProjects.at( index.row() ).path ? mRecentProjects.at( index.row() ).title : QString( tr( "- untitled -" ) ) ).arg( mRecentProjects.at( index.row() ).path );
```

Comment:
@m-kuhn , I was adding the html styling here to avoid having to reach the mRecentProjects object from within the delegate code. I.e., as far as I can make it, we'd have to add a pointer to the mRecentProjects object within the QgsWelcomePageItemDelegate class, which I initially thought wasn't worth the extra lines of code.

How would you go at it?